@re-/assert/test/externalSnapshots.test.ts

```diff
@@ -0,0 +1,105 @@
+import { AssertionError, strict, throws } from "node:assert"
+import { join } from "node:path"
+import { dirName, readJson, writeJson } from "@re-/node"
+import { assert } from "../src/assert.js"
+const testDir = dirName()
+const testFile = "externalSnapshots.test.ts"
+const o = { re: "do" }
+
+describe("Snapshots Using Files", () => {
```

Comment:
Thanks for splitting these up, much better than the monolithic atrocity I was creating 😍